master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Felix Configuration Resource (FelixConfiguration)
+---
+
+A Felix configuration resource (FelixConfiguration) represents Felix configuration options for the cluster.
+
+For `calicoctl` commands that specify a resource type on the CLI, the following
+aliases are supported (all case insensitive): `felixconfiguration`, `felixconfig`, `felixconfigurations`, `felixconfigs`.
+
+See [Configuring Felix]({{site.baseurl}}/{{page.version}}/referance/felix/configuration) for more details.
+
+### Sample YAML
+
+```yaml
+apiVersion: projectcalico.org/v2
+kind: FelixConfiguration
+metadata:
+  name: default
+spec:
+  ipv6Support: false
+  ignoreLooseRPF: true
+  chainInsertMode: Append
+```
+
+### BGP Peer Definition
+
+#### Metadata
+
+| Field       | Description                 | Accepted Values   | Schema |
+|-------------|-----------------------------|-------------------|--------|
+| name     | Unique name to describe this resource instance. Must be specified. | Alphanumeric string with optional `.`, `_`, `-`, or `/` | string |
+
+- Resource with the name "default" has a specific meaning - this contains the Felix global default configuration overrides.
+- Resource with the name "node.<nodename>" has the specific meaning that this contains the node-specific overrides, and will be applied to the node <nodename>.  When deleting a node the FelixConfiguration resource associated with the node will also be deleted.
+
+#### Spec
+
```

Comment:
I do not know if these are the same as what we already have in the Felix config docs....but if so, it might be nice to use an include instead...if possible.

Reply:
these are fields of the config object, Felix config page explains those things primarily as ENV vars, and here we are just listing all the resources and their fields (can be used understanding Calico APIs or creating the config yaml/json files)